cast/js/src/main/resources/preamble.js

```diff
@@ -133,21 +133,26 @@ DOMNode = function DOMNode() { // An impostor for the Node class
         
         this.selectNodes = function(a) {
 	}
+
+	this.hasAttribute = function(a) {
+	}
 };
 
-DOMNode.prototype.addEventListener = function Node_prototype_addEventListener(name, fn) { fn(); };
+DOMNode.prototype.addEventListener = function Node_prototype_addEventListener(name, fn, options) { fn(); };
 
-DOMNode.prototype.removeEventListener = function Node_prototype_removeEventListener(name) {};
+DOMNode.prototype.removeEventListener = function Node_prototype_removeEventListener(name, fn, options) {};
```

Comment:
Same question as previous comment.  In general there are several changes that just fix the number of parameters to be passed in, which are fine, but I'd like to understand how they impact constructed call graphs

Reply:
No impact on edges